GVRf/Framework/jni/engine/renderer/renderer.cpp

```diff
@@ -160,6 +162,49 @@ void Renderer::renderCamera(Scene* scene, Camera* camera, int framebufferId,
     // bone/weight/joint and other assimp data, we will put general model conversion
     // on hold and do this kind of conversion fist
 
+    //  LOGI(" Render Camera Framebuffer %d ", framebufferId);
+
+    DirectionalLight *cameraLight = scene->getDirectionalLight();
+
+    bool renderShadow = cameraLight; // TODO reader from scene;
+
+    if ((framebufferId != 0) && !renderShadow) {
```

Comment:
We render to the non-shadow map rendering mode only if both of these checks are true. I understand the check for !renderShadow. Why do we have the other check?